tests/dummy/app/components/avatar-initials/template.hbs

```diff
@@ -3,17 +3,17 @@
     <div class="col-sm-4">
       <div class="text-center">
         <p class="lead"><code>ember-initials</code></p>
-        {{ember-initials class="img-circle"
-          size=size
-          name=name
-          seedText=seedText
-          defaultName=defaultName
-          textColor=textColor
-          fontSize=fontSize
-          fontWeight=fontWeight
-          fontFamily=fontFamily
-          image=image
-        }}
+        <EmberInitials class="img-circle"
+          @size={{size}}
```

Comment:
Use of `this.` for all the values? So `@size={{this.size}}` etc

Reply:
Demos will be rewritten in the future :)